database/migrations/2019_02_06_012418_set_user_nullable_fields.php

```diff
@@ -20,7 +20,7 @@ public function up()
             $table->boolean('banned')->default(false)->change();
             $table->boolean('is_admin')->default(false)->change();
             $table->boolean('active')->default(false)->change();
-            $table->datetime('last_connection')->default(DB::raw('CURRENT_TIMESTAMP'))->change();
+            $table->datetime('last_connection')->nullable()->change();
```

Comment:
This was a HACK. Is this really the right solution?
Are we sure that having this as nullable now, is not going to break anything?